examples/text_labels_and_annotations/tex_demo.py

```diff
@@ -1,35 +1,34 @@
 """
-=================================
-Rendering math equation using TeX
-=================================
+========
+Tex Demo
+========
 
 You can use TeX to render all of your matplotlib text if the rc
 parameter text.usetex is set.  This works currently on the agg and ps
 backends, and requires that you have tex and the other dependencies
 described at http://matplotlib.org/users/usetex.html
 properly installed on your system.  The first time you run a script
 you will see a lot of output from tex and associated tools.  The next
-time, the run may be silent, as a lot of the information is cached.
-
-Notice how the the label for the y axis is provided using unicode!
+time, the run may be silent, as a lot of the information is cached in
+~/.tex.cache
```

Comment:
This should not be here; I think you may have resolved conflicts in the opposite direction.